app/src/androidTest/java/com/android/bookswap/ui/books/add/ISBNAddTest.kt

```diff
@@ -84,21 +112,31 @@ class ISBNAddTest : TestCase() {
     Assert.assertEquals(
         "12845", isbnField.fetchSemanticsNode().config[SemanticsProperties.EditableText].text)
   }
+  /*
```

Comment:
Do you need to maintain all these comments? They seem awfully large.

Reply:
Yes, because this is a test that has been broken with a merge and i need to talk to the concerned about it during our meeting tomorow.